.env.example

```diff
@@ -1,3 +1,4 @@
 PORT=
 GITHUB_TOKEN=
 GITHUB_WEBHOOK_SECRET=
+ACCEPT_ACTION=
```

Comment:
if we do merge this in, lets remove from this file. this would [force everyone to define this](https://github.com/rolodato/dotenv-safe#dotenv-safe), so i'd rather allow them to use the default when not specified.